packages/tezos-ts-rpc/src/tezos-ts-rpc.ts

```diff
@@ -364,6 +366,32 @@ export class RpcClient {
     };
   }
 
+  /**
+   *
+   * @param options contains generic configuration for rpc calls
+   * @param args contains optional query arguments
+   *
+   * @description Retrieves the list of delegates allowed to bake a block.
+   *
+   * @see https://tezos.gitlab.io/mainnet/api/rpc.html#get-block-id-helpers-baking-rights
+   */
+  async getBakingRights(
+    { block }: RPCOptions = defaultRPCOptions,
+    args: BakingRightsQueryArguments = {}
```

Comment:
Do you think it is reasonable to swap these parameter to follow the same convention as other method?

Reply:
Sure